### PR TITLE
Features/assign task info ui amendments

### PR DESCRIPTION
**Pull request by 247benji**
### Portal

- Added display names to models.
- Moved Notes to right column.
- Moved Source Type and Workspace Affected to left column.
- Fixed alignment of notes column to align to top and bottom of left column.

### CODE REVIEW

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml [hunk 1]
```diff
@@ -11,32 +11,30 @@
     </h6>
     <div class="col-12">
         <div class="card-body pb-0 pt-0">
-            <div class="row justify-content-between">
-                <div class="col-5">
+            <div class="row">
+                <div class="col-12 col-sm-12 col-md-12 col-lg-5 col-xl-5">
```
Comment (UKHOGreg) [inline src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml:15]: ` <div class="col-12 col-sm-12 col-md-12 col-lg-5 col-xl-5">`

Really cool that you have considered responsive design!

One tip though, you don't have to specify all breakpoints:

[https://getbootstrap.com/docs/4.1/layout/grid/](https://getbootstrap.com/docs/4.1/layout/grid/)
> Grid breakpoints are based on minimum width media queries, meaning they apply to that one breakpoint and all those above it (e.g., .col-sm-4 applies to small, medium, large, and extra large devices, but not the first xs breakpoint).

So:
` <div class="col-12 col-sm-12 col-md-12 col-lg-5 col-xl-5">`
could be
` <div class="col-12 col-xl-5">`
I believe? But try it for yourself!

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml [hunk 2]
```diff
@@ -11,32 +11,30 @@
...
                         <div class="col-5">@Html.LabelFor(model => model.PrimaryAssignedTask.WorkspaceAffected, new { @class = "col-form-label" })</div>
                         <div class="col-7">@Html.TextBoxFor(model => model.PrimaryAssignedTask.WorkspaceAffected, new { @class = "form-control assignTaskWorkspaceAffected" })</div>
                     </div>
                 </div>
-            </div>
-            <div class="form-row justify-content-start">
-                <div class="form-group col-12">
-                    <div>@Html.LabelFor(model => model.PrimaryAssignedTask.Notes)</div>
-                    <div>@Html.TextAreaFor(model => model.PrimaryAssignedTask.Notes, htmlAttributes: new { @class = "form-control assignTaskNotes" })</div>
+                <div class="col-12 col-sm-12 col-md-12 col-lg-7 col-xl-7 d-flex flex-column">
```
Comment (UKHOGreg) [inline src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml:33]: See first comment on file!

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml [hunk 3]
```diff
@@ -54,36 +52,34 @@
             </h6>
             <div class="col-12">
                 <div class="card-body pb-0 pt-0">
-                    <div class="row justify-content-between">
-                        <div class="col-5">
+                    <div class="row">
+                        <div class="col-12 col-sm-12 col-md-12 col-lg-5 col-xl-5">
```
Comment (UKHOGreg) [inline src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml:56]: See first comment on file!

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml [hunk 4]
```diff
@@ -54,36 +52,34 @@
...
                                 <div class="col-5">@Html.LabelFor(model => model.AdditionalAssignedTasks.ElementAt(i).WorkspaceAffected, new { @class = "col-form-label" })</div>
                                 <div class="col-7">@Html.TextBoxFor(model => model.AdditionalAssignedTasks.ElementAt(i).WorkspaceAffected, new { @class = "form-control assignTaskWorkspaceAffected" })</div>
                             </div>
                         </div>
-                    </div>
-                    <div class="form-row justify-content-start">
-                        <div class="form-group col-12">
-                            <div>@Html.LabelFor(model => model.AdditionalAssignedTasks.ElementAt(i).Notes)</div>
-                            <div>@Html.TextAreaFor(model => model.AdditionalAssignedTasks.ElementAt(i).Notes, htmlAttributes: new { @class = "form-control assignTaskNotes" })</div>
+                        <div class="col-12 col-sm-12 col-md-12 col-lg-7 col-xl-7 d-flex flex-column">
```
Comment (UKHOGreg) [inline src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml:74]: See first comment on file!

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml (hunk 1 above)
Comment (247benji) [reply src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml]: Removed unnecessary breakpoints.

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml (hunk 2 above)
Comment (247benji) [reply src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml]: Removed unnecessary breakpoints.

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml (hunk 3 above)
Comment (247benji) [reply src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml]: Removed unnecessary breakpoints.

File: src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml (hunk 4 above)
Comment (247benji) [reply src/Portal/Portal/Pages/DbAssessment/_AssignTask.cshtml]: Removed unnecessary breakpoints.

**Pull request merged**